lib/classes/Service.js

```diff
@@ -147,6 +147,9 @@ class Service {
     this.configValidationMode = serverlessFile.configValidationMode || 'warn';
     this.disabledDeprecations = serverlessFile.disabledDeprecations;
 
+    // todo: add `|| 'error'` in new major - don't default to warn to indicate the deprecation message should be printed
```

Comment:
Let's remove this comment (I think with new major we will just change the way we handle this property , and do not change anything here)